docs/filtering.md

```diff
@@ -54,9 +54,9 @@ on the datapoint that will not affect matching.
 
 The negation of items in either `metricNames` or a `dimensions` map value list
 item will serve to override (and thus reinclude) only other already excluded
-items from that same list.  Thus, if you want to do "whitelisting" of metric
-names or dimension values.hi, you can provide a list like `[ '*',
-'!whitelisted1', '!whitelisted2' ]` to exclude everything but those two
+items from that same list.  Thus, if you want to filter metric
+names or dimension values, you can provide a list like `[ '*',
+'!allowlist1', '!allowlist2' ]` to exclude everything but those two
```

Comment:
Since we already use the verbage `metricsToInclude` and `metricsToExclude` in regards to filtering, I think `inclusionList` would read better here.